src/test/java/io/iron/ironmq/IronMQTest.java

```diff
@@ -657,24 +657,10 @@ public void testUpdateQueuePushParameters() throws IOException {
     }
 
     /**
-     * This test shows how to update alerts of a queue
      * Expected that:
-     * - new alert will be available after update
      * @throws IOException
      */
     @Test
-    public void testUpdateQueueAlerts() throws IOException {
-        queue.create();
-        ArrayList<Alert> alerts = new ArrayList<Alert>();
-        alerts.add(new Alert(Alert.typeProgressive, Alert.directionAscending, 5, "some_q"));
-        QueueModel info = queue.updateAlerts(alerts);
-
-        Assert.assertEquals(5, info.getAlerts().get(0).getTrigger());
-        Assert.assertEquals(Alert.directionAscending, info.getAlerts().get(0).getDirection());
-        Assert.assertEquals(Alert.typeProgressive, info.getAlerts().get(0).getType());
-        Assert.assertEquals("some_q", info.getAlerts().get(0).getQueue());
-    }
-
```

Comment:
@Ayodejiola If you delete the whole method - you should delete all the headers and lines. In this case you should delete all the lines between 659 and 677